less/lists.less

```diff
@@ -381,14 +381,23 @@
 
 .section.associations, .view-details-associations, .list {
   .list-item {
-    flex-basis: 48%;
-    @media (min-width: @large-width){
-      flex-basis: 32%;
+    font-size: 90%;
```

Comment:
Are ``%`` the good way to set a font size? I though ``em`` were recommended instead.